src/status_im/contexts/communities/actions/channel_view_details/view.cljs

```diff
@@ -0,0 +1,113 @@
+(ns status-im.contexts.communities.actions.channel-view-details.view
+  (:require [quo.core :as quo]
+            [quo.foundations.colors :as colors]
+            [quo.theme :as quo.theme]
+            [react-native.core :as rn]
+            [status-im.common.contact-list-item.view :as contact-list-item]
+            [status-im.common.contact-list.view :as contact-list]
+            [status-im.common.home.actions.view :as home.actions]
+            [status-im.contexts.communities.actions.channel-view-details.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn contact-item-render
+  [public-key]
+  (let [show-profile-actions          #(rf/dispatch [:show-bottom-sheet
+                                                     {:content (fn [] [home.actions/contact-actions
+                                                                       {:public-key public-key}])}])
+        [primary-name secondary-name] (rf/sub [:contacts/contact-two-names-by-identity public-key])
+        {:keys [ens-verified added?]} (rf/sub [:contacts/contact-by-address public-key])]
+    [contact-list-item/contact-list-item
+     {:on-press      #(rf/dispatch [:chat.ui/show-profile public-key])
+      :on-long-press show-profile-actions
+      :accessory     {:type     :options
+                      :on-press show-profile-actions}}
+     {:primary-name   primary-name
+      :secondary-name secondary-name
+      :public-key     public-key
+      :ens-verified   ens-verified
+      :added?         added?}
+     {:public-key public-key}]))
+
+(defn footer
+  []
+  [rn/view {:style style/footer}])
+
+(defn members
+  [items]
+  [rn/section-list
+   {:key-fn                            :public-key
+    :content-container-style           {:padding-bottom 20}
+    :get-item-layout                   (fn [_ index]
+                                         #js
+                                          {:length 200
+                                           :offset (* 200 index)
+                                           :index  index})
+    :content-inset-adjustment-behavior :never
+    :sections                          items
+    :sticky-section-headers-enabled    false
+    :render-section-header-fn          contact-list/contacts-section-header
+    :render-section-footer-fn          footer
+    :render-fn                         contact-item-render
+    :scroll-event-throttle             8}])
+
+(defn f-view
+  [{:keys [chat-id community-id theme]}]
+  (let [{:keys [description chat-name emoji muted
+                chat-type]
+         :as   chat}  (rf/sub [:chats/chat-by-id
+                               chat-id])
+        pins-count    (rf/sub [:chats/pin-messages-count chat-id])
+        items         (rf/sub [:communities/sorted-community-members-section-list
```

Comment:
You are adding subscriptions outside the renderer, which makes the component non-reactive. Most certainly a bug to the user. If anything this component needs gets updated in the app db, it won't re-render. Saving very specific use cases, subscriptions should be called inside the renderer.

Reply:
Apparently, I forgot to push the last commit :)
Also I had fixed the text for the muted/unmuted status there, Here it was hardcoded to mute channel